.travis.yml

```diff
@@ -9,8 +9,10 @@ matrix:
       addons: {apt: {packages: [cabal-install-1.16,ghc-7.6.3], sources: [hvr-ghc]}}
     - env: CABALVER=1.18 GHCVER=7.8.4
       addons: {apt: {packages: [cabal-install-1.18,ghc-7.8.4], sources: [hvr-ghc]}}
-    - env: CABALVER=1.22 GHCVER=7.10.1
-      addons: {apt: {packages: [cabal-install-1.22,ghc-7.10.1],sources: [hvr-ghc]}}
+    - env: CABALVER=1.22 GHCVER=7.10.2
```

Comment:
I'm confused , I definitely have 7.10.3 in my own Travis build matrices.
My monad-STE repo for example.

On Sunday, May 29, 2016, Oleg Grenrus <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/ekmett/bound/pull/24#discussion_r65008551>:
>
> > @@ -9,8 +9,10 @@ matrix:
> >        addons: {apt: {packages: [cabal-install-1.16,ghc-7.6.3], sources: [hvr-ghc]}}
> >      - env: CABALVER=1.18 GHCVER=7.8.4
> >        addons: {apt: {packages: [cabal-install-1.18,ghc-7.8.4], sources: [hvr-ghc]}}
> > -    - env: CABALVER=1.22 GHCVER=7.10.1
> > -      addons: {apt: {packages: [cabal-install-1.22,ghc-7.10.1],sources: [hvr-ghc]}}
> > +    - env: CABALVER=1.22 GHCVER=7.10.2
>
> We can either
>
>    - use ghc-7.10.2
>    - use cabal-install-1.24
>    - or compile cabal-install depending on newer Cabal
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ekmett/bound/pull/24/files/e1606a77c2ba20f7e63657078522859b2fec30c6#r65008551>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAQwufihKgwnbcAjyRx1A-5-dnCTxCbks5qGc02gaJpZM4IpUdR>
> .
>